astro.config.mjs

```diff
@@ -8,6 +8,8 @@ import preact from "@astrojs/preact";
 
 // https://astro.build/config
 export default defineConfig({
+  site: 'https://tcet-opensource.github.io',
```

Comment:
change this to tnp.tcetmumbai.in

Reply:
might have to remove the base parameter in case using a custom URL. Will have to test that out